### ISSUE TITLE
User Page: Layout

### ISSUE THREAD

**Issue by ClaytonAndTheClayTones** [mva3212/code#9]
Design a layout for a page that displays the following as it pertains to a specific user:
-  All farms (and preferably all crops with those farms...collapseable?  Separate tab?)
-  Demographic and personal info
-  All logs in a condensed form, regardless of what they are attached to, filterable via ajax and, by default, sorted by recentness, though other sort options are good.

**Comment by mva3212** [comment]
Good idea.  I'd like to have the UI express the relation between farms and journals farms and crops as some kind of containment.  

All crops either belong to a user created farm or the user's default farm, all journals belong to a farm or a crop or the users default farm.  I was thinking having a default "farm" thumbnail that when hovered over showed the crops and journals inside, clicking on it would bring you into that farm.  The user could of course, upload a picture of their farm that would replace the default picture.  

I think in the beginning to keep it simple we create a farm, by default, for every new user and only give them the option of updating and customizing their default farm.  Creating additional farms is something I'd like to keep off the table in early revisions, for simplicity sake.  We of course will want to offer the ability to create additional farms in the future, but possibly only to subscribers and early adopters.

I think we should do the same thing for crops and journals in  the beginning as it will simplify a lot of things for us.  Of course we should have everything in place to implement these features easily.
